app/controllers/choices_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ChoicesController < ApplicationController
+
+  def create
+    @question = Question.find(params[:question])
+    text = params[:text]
+    correct = params[:correct] == '1'
+    Choice.create(:text => text, :correct => correct, :question_id => @question.id)
```

Comment:
a good practice is to have strong params to create new record. EX https://github.cerner.com/HealtheLife/education_library/blob/master/app/controllers/media_controller.rb#L28-L37

Reply:
what if any param is missing?